docs/modeling-your-data/modeling-your-data-with-dbt/dbt-quickstart/fractribution/index.md

```diff
@@ -89,6 +95,37 @@ Execute the following either through your CLI, within dbt Cloud, or within [Snow
 dbt run --select snowplow_fractribution
 ```
 ### 5. Run the python script to generate the final models
+
+Depending on your setup, please follow the appropriate steps below. All these methods will create the following tables in your warehouse:
+
+- `snowplow_fractribution_path_summary_with_channels`
+- `snowplow_fractribution_report_table`
+- `snowplow_fractribution_channel_attribution`
+
+<details>
+<summary>Run on Snowflake using Snowpark</summary>
+
+If you are using Snowflake, you can have the dbt model run the python scripts for you using their Snowpark features. To enable these you simply need to set the `snowplow__run_python_script_in_snowpark` and `snowplow__attribution_model_for_snowpark` variables and run the model.
+
```

Comment:
Can we change "If you are using Snowflake, you can have the dbt model run the python scripts for you using their Snowpark features." 
to something like:
"If you are using Snowflake, you can automatically run the python scripts using Snowpark when running the dbt package. This is done using macros that create and run a stored procedure on Snowpark after the dbt models have completed."
since the way they work isn't really based on using their snowpark features necessarily